msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -295,6 +295,64 @@ The JSON response body for Meeting Details API is as follows:
     }
 } 
 ```
+
+## CART API 
+
+The communication access real-time translation (CART) API enables your app to expose a POST endpoint for Microsoft Teams CART captions.
+When end users enable captions in a Microsoft Teams meeting, the CART API displays the text content sent to the endpoint.
+
+### Base URL
```

Comment:
I'd change Base URL to CART URL here and in the paragraph, since Base URL refers to the common part of the URL